packages/modules/itwin-viewer-react/src/services/Initializer.ts

```diff
@@ -134,6 +134,11 @@ class Initializer {
     return this._initialized;
   }
 
+  /** expose initialized cancel method */
```

Comment:
In hindsight, I have some concerns about canceling the promise. Basically this promise handles initialization of the various iModel.js packages, each require to be initialized once and only once. If we cancel the promise after some are initialized but not all, wouldn't the next time around cause errors? 

One of the changes that I am making in the new package is instead of the "initialized" property and managing initialization within the Initializer, I am just checking UICore.initialized and returning if true. Still could create an issue like I described above, but at least we would not rely on the initializer to maintain that state. 

Of course all of this assumes that you are passing the same parameters for each "instance"